__docs__/wonder-blocks-typography/typography.stories.tsx

```diff
@@ -73,34 +77,47 @@ export const ControlProps: StoryObj<typeof Title> = {
     },
 };
 
-export const TypographyElements: StoryObj<any> = () => (
-    <View>
-        <Title>Title</Title>
-        <HeadingLarge>HeadingLarge</HeadingLarge>
-        <HeadingMedium>HeadingMedium</HeadingMedium>
-        <HeadingSmall>HeadingSmall</HeadingSmall>
-        <HeadingXSmall>HeadingXSmall</HeadingXSmall>
-        <BodySerifBlock>BodySerifBlock</BodySerifBlock>
-        <BodySerif>BodySerif</BodySerif>
-        <BodyMonospace>BodyMonospace</BodyMonospace>
-        <Body>Body</Body>
-        <LabelLarge>LabelLarge</LabelLarge>
-        <LabelMedium>LabelMedium</LabelMedium>
-        <LabelSmall>LabelSmall</LabelSmall>
-        <LabelXSmall>LabelXSmall</LabelXSmall>
-        <Tagline>Tagline</Tagline>
-        <Caption>Caption</Caption>
-        <Footnote>Footnote</Footnote>
-    </View>
-);
+/**
+ These are all the available Thunderblocks typography elements with their names
+ written out in their respective styles. Wrapping them in `ThemeSwitcher` with
+ `theme ="thunderblocks"` will include the Plus Jakarta Sans typeface, otherwise
+ they will default to Lato.
+ */
+export const NewTypographyElements: StoryObj<any> = {
+    render: () => (
+        <View>
+            <Heading size="xxLarge">Heading</Heading>
+            <BodyText>BodyText</BodyText>
+        </View>
+    ),
+    globals: {theme: "thunderblocks"},
+};
 
-TypographyElements.parameters = {
-    docs: {
-        description: {
-            story: `These are all the available typography elements
-            with their names written out in their respective styles.`,
-        },
-    },
+/**
+These are all the available classic typography elements with their names written
+out in their respective styles.
+ */
+export const ClassicTypographyElements: StoryObj<any> = {
+    render: () => (
+        <View>
+            <Title>Title</Title>
+            <HeadingLarge>HeadingLarge</HeadingLarge>
+            <HeadingMedium>HeadingMedium</HeadingMedium>
+            <HeadingSmall>HeadingSmall</HeadingSmall>
+            <HeadingXSmall>HeadingXSmall</HeadingXSmall>
+            <BodySerifBlock>BodySerifBlock</BodySerifBlock>
+            <BodySerif>BodySerif</BodySerif>
+            <BodyMonospace>BodyMonospace</BodyMonospace>
+            <Body>Body</Body>
+            <LabelLarge>LabelLarge</LabelLarge>
+            <LabelMedium>LabelMedium</LabelMedium>
+            <LabelSmall>LabelSmall</LabelSmall>
+            <LabelXSmall>LabelXSmall</LabelXSmall>
+            <Tagline>Tagline</Tagline>
+            <Caption>Caption</Caption>
+            <Footnote>Footnote</Footnote>
```

Comment:
It would also be great to have visual regression tests for the classic typography components for both OG and TB themes (to make sure nothing is looking wonky!)

Reply:
++ to this, you can use Chromatic modes for this (see my previous example). Adding to that, this is probably something that could be covered with a `typography-testing-snapshots` file that uses the `StateSheet` or `AllVariants` custom components.

Reply:
I enabled Chromatic modes for this story and the SizesAndWeights stories. Do you think that is sufficient for covering the components for regressions? I could add an `AllVariants` snapshot for all of the components as a follow-up PR.

Reply:
Probably adding `AllVariants` would be helpful as we started standardizing on the `Testing > [component name] - Snapshots` (and `*-testing-snapshots.stories.tsx` file) to know when we want to cover different variants of our components in Chromatic. Follow-up PR sounds great!